SimGeneral/PreMixingModule/python/mixOne_premix_on_sim_cfi.py

```diff
@@ -250,3 +252,167 @@
+            pileInputTag = "simSiPixelDigis:Pixel",
+        ),
+        phase2OTSimLink = cms.PSet(
+            workerType = cms.string("PreMixingPixelDigiSimLinkWorker"),
+            #
+            labelSig = cms.InputTag("simSiPixelDigis:Tracker"),
+            pileInputTag = cms.InputTag("simSiPixelDigis:Tracker"),
+            collectionDM = cms.string("Phase2OTDigiSimLink"),
+        ),
+    ),
+)
+
+# ECAL
+phase2_ecal.toModify(mixData,
+    workers = dict(
+        ecal = dict(
+            doES = False,
+            EBPileInputTag = "simEcalDigis:ebDigis",
+            EEPileInputTag = "simEcalDigis:eeDigis",
+        )
+    )
+)
+phase2_hgcal.toModify(mixData, workers=dict(ecal=dict(doEE=False)))
+
+# HCAL
+phase2_hcal.toModify(mixData,
+    workers = dict(
+        hcal = dict(
+            HBHEPileInputTag = "simHcalDigis",
+            HOPileInputTag = "simHcalDigis",
+            HFPileInputTag = "simHcalDigis",
+            QIE10PileInputTag = "simHcalDigis:HFQIE10DigiCollection",
+            QIE11PileInputTag = "simHcalDigis:HBHEQIE11DigiCollection",
+            ZDCPileInputTag = "simHcalUnsuppressedDigis",
+        )
+    )
+)
+
+phase2_hgcal.toModify(mixData,
+    workers = dict(
+        hgcee = cms.PSet(
+            hgceeDigitizer,
+            #
+            workerType = cms.string("PreMixingHGCalWorker"),
+            #
+            digiTagSig = cms.InputTag("mix", "HGCDigisEE"),
+            pileInputTag = cms.InputTag("mix", "HGCDigisEE"),
+        ),
+        hgchefront = cms.PSet(
+            hgchefrontDigitizer,
+            #
+            workerType = cms.string("PreMixingHGCalWorker"),
+            #
+            digiTagSig = cms.InputTag("mix", "HGCDigisHEfront"),
+            pileInputTag = cms.InputTag("mix", "HGCDigisHEfront"),
+        ),
+        hgcheback = cms.PSet(
+            hgchebackDigitizer,
+            #
+            workerType = cms.string("PreMixingHGCalWorker"),
+            #
+            digiTagSig = cms.InputTag("mix", "HGCDigisHEback"),
+            pileInputTag = cms.InputTag("mix", "HGCDigisHEback"),
+        ),
+        caloTruth = cms.PSet(
+            workerType = cms.string("PreMixingCaloParticleWorker"),
+            #
+            labelSig = cms.InputTag("mix", "MergedCaloTruth"),
+            pileInputTag = cms.InputTag("mix", "MergedCaloTruth"),
+            collectionDM = cms.string("MergedCaloTruth"),
+        )
+    )
+)
+
+# Muon
+phase2_muon.toModify(mixData,
+    workers = dict(
+        dt = dict(pileInputTag = "simMuonDTDigis"),
+        rpc = dict(pileInputTag = "simMuonRPCDigis"),
+        csc = dict(
+            strip = dict(pileInputTag = "simMuonCSCDigis:MuonCSCStripDigi"),
+            wire = dict(pileInputTag = "simMuonCSCDigis:MuonCSCWireDigi"),
+            comparator = dict(pileInputTag = "simMuonCSCDigis:MuonCSCComparatorDigi"),
+        ),
+        gem = cms.PSet(
+            workerType = cms.string("PreMixingGEMWorker"),
+            #
```

Comment:
is there a reason for the blank comments here?

Reply:
Following the definition of `mixData` above aiming to improve readability. On the other hand the same can be achieved with empty lines (I probably used comments in #22416 because the `DataMixingModule` configuration was full of empty comments), and probably for these workers with only one workerType, signal input, pileup input, and output instance label the empty line is not really needed at all.

Would you like to get them cleaned up in this PR (also in the definition of `mixData`)?

Reply:
Ah, I hadn't noticed it above. Yes, to me this is unnecessary for most of the PSets with only a few parameters. The cleanup could also be done all together in a subsequent PR.